@types/match-url-wildcard/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'match-url-wildcard' {
```

Comment:
Move typings to the module. If it's not possible create a separate module `@types/<module_name>` and publish it.